### DISCUSSION TITLE
Nowhere to report this

### DISCUSSION THREAD

**Discussion by lokman2k5** [elementary/triage#635] [Q&A]
Hi, in Web, there are bugs with text selection when using touch. The video below explains. But simply selecting text is really buggy and the bugs happen randomly, long pressing on text sometimes selects text sometimes not, sometimes opens context menus sometimes not. The preferred behavior is to select text and open the context menu at the same time.

https://github.com/user-attachments/assets/24496521-4bbd-4ac8-8402-bdad9f3fc25b

eos 8, Wayland, Thinkpad X1 yoga gen 3.

**Comment by danirabbit** [comment]
Please open an issue report at https://github.com/elementary/browser/issues thanks!
